### PR TITLE
Security advisories and Octopus products

### PR DESCRIPTION
**Pull request by wordlee**
A blog post about how we're changing the way we deliver security advisories

### CODE REVIEW

File: blog/2021-02/security-advisories/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+In terms of notifying our customers of security vulnerabilities we have always assumed our customers are either reading the release notes or are periodically checking CVE databases for any security vulnerabilities. However, we believe we can do more to help our customers have the information they need to ensure their environments remain secure.
+
+To address this we're introducing Octopus Security Advisories.
+
+## Octopus security advisories
+
+Octopus security advisories are essentially the aggregation of the release notes, CVE details, and extra information we want to provide to our customers in one place so customers don't have to go hunting for information.
+
+Some of the information provided in the security advisories will be:
+- Vulnerability risk based on an Octopus assessment (more to come on this).
```
Comment (wizedkyle) [inline blog/2021-02/security-advisories/index.md:33]: Can we change this line to "vulnerability severity based on an Octopus Deploy assessment" ?

File: blog/2021-02/security-advisories/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+bannerImage: blogimage-security.png
+tags:
+  - Trust & Security
+  - Product
+  - Advisories
+---
+
+![A stylized Octopus & Shield icon](blogimage-security.png)
+
+It's no surprise that software products have security vulnerabilities, or that when they do occur, it's important for vendors to have effective communications strategies to ensure systems are secure and users aren't left vulnerable.
```
Comment (wizedkyle) [inline blog/2021-02/security-advisories/index.md:16]: Should it be communication or communications?

**Review by wizedkyle — CHANGES_REQUESTED** [review]
Just a question about wording, and a change for once sentence

File: blog/2021-02/security-advisories/index.md (hunk 1 above)
Comment (wordlee) [inline blog/2021-02/security-advisories/index.md:33]:
```suggestion
- Vulnerability severity based on an Octopus Deploy assessment (more to come on this).
```

File: blog/2021-02/security-advisories/index.md (hunk 2 above)
Comment (wordlee) [inline blog/2021-02/security-advisories/index.md:16]:
```suggestion
It's no surprise that software products have security vulnerabilities, or that when they do occur, it's important for vendors to have effective communication strategies to ensure systems are secure and users aren't left vulnerable.
```

File: blog/2021-02/security-advisories/index.md (hunk 1 above)
Comment (wizedkyle) [reply blog/2021-02/security-advisories/index.md:33]: No, we can remove this now as we have the severity levels guide on the site

**Review by wizedkyle — APPROVED** [review]
LGTM :shipit:

**Pull request merged**